test/unit/sprites_rendered-target.js

```diff
@@ -1,6 +1,66 @@
 var test = require('tap').test;
 var RenderedTarget = require('../../src/sprites/rendered-target');
 var Sprite = require('../../src/sprites/sprite');
+var Runtime = require('../../src/engine/runtime');
+
+
+var fakeRenderer = function (){
```

Comment:
I feel like the "fakeRenderer" might be better split out and added as a module to "test/fixtures". Thoughts?